rust/src/battery.rs

```diff
@@ -0,0 +1,178 @@
+use crate::acpi::{Acpi, AcpiEvalOutputBufferV1};
+
+use ratatui::{
+    buffer::Buffer,
+    layout::Rect,
+    style::{Color, Stylize, palette::tailwind},
+    text::Line,
+    widgets::{Block, Borders, Gauge, Padding, Paragraph, Widget},
+};
+
+const BATGAUGE_COLOR: Color = tailwind::ORANGE.c800;
+const LABEL_COLOR: Color = tailwind::SLATE.c200;
+
+#[derive(Default)]
+pub struct BstData {
```

Comment:
What does `Bst` stand for here? Definitely not binary search tree? Perhaps Battery State T...?